src/components/HealthCardDisplay/HealthCardDisplay.js

```diff
@@ -339,11 +379,47 @@ const HealthCardDisplay = () => {
                       {patientData.immunizations.map((item) => (
                         <div key={item.fullUrl}>
                           <ListItem>
-                            <HealthCardVaccination immunization={item.resource} />
+                            <HealthCardVaccination
+                              immunization={item.resource}
+                            />
                           </ListItem>
                         </div>
                       ))}
                     </List>
+                    <Grid
+                      container
+                      xs={10.5}
```

Comment:
This is causing a console error "Failed prop type: The prop `xs` of `Grid` can only be used together with the `item` prop."